katello/katello-backup

```diff
@@ -40,14 +40,22 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
+def run_cmd(command)
+  `#{command}`
+  if $?.exitstatus != 0
+    puts "Failed '#{command}'"
```

Comment:
Overall the change looks good to me, I'd prefer if we did `STDERR.puts` here instead (and everywhere else we print an error and exit).

Some sysadmins have cron send stdout to /dev/null, but keep stderr so they get a mail from cron when the job fails that has the error message in it.